auslib/blobs/base.py

```diff
@@ -161,3 +161,10 @@ def getFooterXML(self):
         """
         footer = '</updates>'
         return footer
+
+    def getReferencedReleases(self):
+        """
+        :return: Returns set of names of partially referenced releases that the current
+        release references
+        """
+        return NotImplementedError()
```

Comment:
As is, this will cause errors whenever we try to extract non-AppRelease blobs from the database. This should either return an empty set, or all other blobs should implement this and return an empty set.